constraint/pkg/client/crd_helpers_test.go

```diff
@@ -103,6 +103,7 @@ func tProp(t string) apiextensions.JSONSchemaProps {
 }
 
 func expectedSchema(pm propMap) *apiextensions.JSONSchemaProps {
+	pm["enforcementaction"] = apiextensions.JSONSchemaProps{Type: "string"}
```

Comment:
Is this necessary? This is meant to allow building of arbitrary schema, and I don't see "enforcementAction" tested anywhere?

Reply:
this was necessary as the tests were failing because `createSchema` now adds `enforcementAction` to `spec` by default, but `expectedSchema` calls `prop` which does not include `enforcementAction`.
https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/client/crd_helpers_test.go#L93-L98

Reply:
ack, SGTM